source/physics_lists/PetaloPhysics.cc

```diff
@@ -93,4 +102,28 @@ void PetaloPhysics::ConstructProcess()
     pmanager->RemoveProcess(cs);
   }
 
+
+  if (nest_) {
+    PetaloDetector* petalo = new PetaloDetector();
+    G4double e_field = petalo->FitEF(0, 0, 0);
+    PetaloPersistencyManager* pm =
+      dynamic_cast<PetaloPersistencyManager*>(G4VPersistencyManager::GetPersistencyManager());
+    pm->SetElectricField(e_field);
+    NEST::NESTcalc* petaloCalc = new NEST::NESTcalc(petalo);
```

Comment:
What does `Calc` mean?

Reply:
I guess that it stands for "calculations". `NESTcalc` is the class of NEST that calculates the light and charge yields, among other things.